### PR TITLE
Improve the kill children processes

### PR DESCRIPTION
**Pull request by yhwen**
Fixes # .

### Description

Change to use process.kill() to kill the children processes. This force kill the example like flow child process.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/private/fed/app/utils.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def kill_child_processes(parent_pid, sig=signal.SIGTERM):
         return
     children = parent.children(recursive=True)
     for process in children:
-        process.send_signal(sig)
```
Comment (YuanTingHsieh) [inline nvflare/private/fed/app/utils.py:59]: Thanks for the fix, should we remove this "sig" argument since it is not used?

Comment (yhwen) [reply nvflare/private/fed/app/utils.py:59]: removed.

**Comment by YuanTingHsieh** [comment]
@yhwen can you fix the unit test, one import is not needed anymore, thanks

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Pull request merged**